<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstLevel</class>
 <widget class="QWidget" name="firstLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>黄金矿工</string>
  </property>
  <widget class="QLabel" name="oldMan">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/oldMan/photo/oldMan.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="firstMap">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>1200</width>
     <height>650</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/firstLevel/photo/playingBackground.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentMoney">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前金额：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="goalMoney">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>目标金额：$650</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lastingtime">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>剩余时间：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="firstLevel_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>90</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="text">
    <string>第1关</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="drawline" name="invisibleListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
  </widget>
  <widget class="myWidget" name="youwangWidget" native="true">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>108</y>
     <width>146</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="bigGold">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/bigGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="showCurrentMoney">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>$0</string>
   </property>
  </widget>
  <widget class="QLabel" name="remainingTime">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>20</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QLabel" name="addNumber">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(55, 158, 54);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fail">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>560</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bigGold2">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>410</y>
     <width>130</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/bigGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone1">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>300</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/stone/photo/stones.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/stone/photo/stones.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="smallGold3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/golds/photo/smallGold.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stone3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/stone/photo/stones.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="luckBag1">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>560</y>
     <width>70</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/luckyBag/photo/luckyBag.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentBomb">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>炸弹数：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="bomb">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>60</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <zorder>oldMan</zorder>
  <zorder>firstMap</zorder>
  <zorder>currentMoney</zorder>
  <zorder>goalMoney</zorder>
  <zorder>lastingtime</zorder>
  <zorder>firstLevel_2</zorder>
  <zorder>bigGold</zorder>
  <zorder>showCurrentMoney</zorder>
  <zorder>remainingTime</zorder>
  <zorder>addNumber</zorder>
  <zorder>smallGold1</zorder>
  <zorder>smallGold2</zorder>
  <zorder>bigGold2</zorder>
  <zorder>stone1</zorder>
  <zorder>stone2</zorder>
  <zorder>smallGold3</zorder>
  <zorder>stone3</zorder>
  <zorder>invisibleListWidget</zorder>
  <zorder>luckBag1</zorder>
  <zorder>youwangWidget</zorder>
  <zorder>currentBomb</zorder>
  <zorder>bomb</zorder>
  <zorder>fail</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drawline</class>
   <extends>QListWidget</extends>
   <header>drawline.h</header>
  </customwidget>
  <customwidget>
   <class>myWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
